pulsar-io/canal/src/main/java/org/apache/pulsar/io/canal/CanalSource.java

```diff
@@ -0,0 +1,152 @@
+package org.apache.pulsar.io.canal;
```

Comment:
This needs to have Apache headers. You can use `mvn license:format` to automatically add